admin/app/views/layouts/solidus_admin/application.html.erb

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="<%= current_locale %>">
   <head>
+    <%= favicon_link_tag 'favicon.ico' %>
```

Comment:
What is the error here, just for the sake of curiosity? Something related to a11y?

Reply:
No, not accessibility, it's just capybara complaining for the 500 received when trying to fetch the default `/favicon.ico`.